IdentityCore/src/webview/MSIDWebviewSession.h

```diff
@@ -26,26 +26,24 @@
 //------------------------------------------------------------------------------
 
 #import <Foundation/Foundation.h>
+#import "MSIDWebviewNotifiable.h"
 
 @protocol MSIDWebviewInteracting;
+
 @class MSIDWebviewFactory;
 
-@interface MSIDWebviewSession : NSObject
+@interface MSIDWebviewSession : NSObject <MSIDWebviewNotifiable>
 
 @property NSObject<MSIDWebviewInteracting> *webviewController;
 @property MSIDWebviewFactory *factory;
 @property NSString *requestState;
-@property BOOL verifyState;
+@property NSString *redirectUri;
```

Comment:
why did we replace state with redirect uri :)?

Reply:
these are 2 separate things, 
1. verifyState is now mandatory, so no need to have it as a property
2. redirectUri is now a part of a session.